### PR TITLE
Make SplashCookiesMiddleware accept COOKIES_DEBUG

### PR DESCRIPTION
**Pull request by lopuhin**
.. to behave like scrapy CookiesMiddleware. This can be useful for debugging.

Debug methods are copied from it. It would be possible to inherit from it and use the methods from the parent class, but this would be more confusing perhaps.

I though about a good place to mention it in the docs, but could not find it (and maybe it's not significant enough for a special mention).

### CODE REVIEW

**Comment by lopuhin** [comment]
Ah, but the debug methods require adoptation, let me fix them first :)

**Comment by lopuhin** [comment]
Now the output looks the same as for the scrapy middleware (I output only cookie names and values).

File: scrapy_splash/middleware.py [hunk 1]
```diff
@@ -45,8 +46,13 @@ class SplashCookiesMiddleware(object):
...
     after SplashMiddleware.
     """
-    def __init__(self):
+    def __init__(self, debug=False):
         self.jars = defaultdict(CookieJar)
+        self.debug = debug
+
+    @classmethod
+    def from_crawler(cls, crawler):
+        return cls(debug=crawler.settings.getbool('COOKIES_DEBUG'))
```
Comment (kmike) [inline scrapy_splash/middleware.py]: What do you think about naming this option SPLASH_COOKIES_DEBUG?

Comment (lopuhin) [inline scrapy_splash/middleware.py]: Yes, I like it better, especially since there are other settings prefixed with `SPLASH_`, thanks! I updated the PR.

Comment (kmike) [inline scrapy_splash/middleware.py]: Unlike `COOKIES_DEBUG`, `SPLASH_COOKIES_DEBUG` option needs docs.

Comment (lopuhin) [inline scrapy_splash/middleware.py]: Maybe a good place would be to mention all available settings at the end of the Configuration section?

Comment (kmike) [inline scrapy_splash/middleware.py]: A good idea! That'd be nice to have them all in a single place. E.g. there is only a passing mention of SPLASH_LOG_400 in README.

Comment (lopuhin) [inline scrapy_splash/middleware.py]: Yep, agreed! I tried to document them in c06da89e027c9f5e46836c81353360e3822c6165 - I'm not entirely sure about the `SPLASH_LOG_400` description.

Comment (kmike) [inline scrapy_splash/middleware.py]: 👍

**Pull request merged**